components/Video.ts

```diff
@@ -0,0 +1,30 @@
+import { jwt } from 'twilio';
+import { connect } from 'twilio-video';
+
+const getRandomIntInclusive = (min, max) => {
+  min = Math.ceil(min);
+  max = Math.floor(max);
+  return Math.floor(Math.random() * (max - min + 1)) + min;
+};
+
+const generateToken = () => {
+  const twilioAccountSid = process.env.TWILIO_SID;
+  const twilioApiKey = process.env.TWILIO_API_KEY;
+  const twilioApiSecret = process.env.TWILIO_API_SECRET;
+  const token = new jwt.AccessToken(
+    twilioAccountSid,
+    twilioApiKey,
+    twilioApiSecret
+  );
+  // @ts-ignore
```

Comment:
`token.identity` doesn't have a TS definition for some reason, so ignore it.

Reply:
You could try non-null assertion here as well!

Reply:
>You could try non-null assertion here as well!
 
What is that?

Reply:
Oh, I see your comment below. Thanks!

Reply:
Hmm, the error is:

>2339: Property 'identity' does not exist on type 'AccessToken'.

I'm not sure that a non-null assertion will fix it?